APIManager.js

```diff
@@ -0,0 +1,67 @@
+//This is the class that will manage all your APIs
+class APIManager {
+    constructor() {
+      this.data = {};
+    }
+    loadData() {
+      //get user
+      $.ajax({
+        method: "GET",
+        url: "https://randomuser.me/api/?results=7&inc=name,location,picture",
+        dataType: "json",
+        success: (data) => {
+          this.data["user"] = data.results[0];
+          // render.renderUser({ user: data.results[0] });
+          data.results.splice(0, 1);
+          const friends = data.results.map((d) => {
```

Comment:
you cold have this all on one line so you don't need a return 
`const friends = data.results.map( d => {name: d.name})`